### PR TITLE
[hi] Localize continuous_delivery.md

### PR DESCRIPTION
**Pull request by verma-kunal**
Signed-off-by: Kunal Verma <vkunal321@gmail.com>

Localizes `content/hi/continuous-delivery.md` to Hindi

This is an intial PR as may need some improvements!

Tracking issue: #270

### CODE REVIEW

File: content/hi/continuous_delivery.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: निरंतर डिलीवरी (CD)
+status: Completed
+category: concept
+---
+
+## यह क्या है 
+निरंतर डिलीवरी, जिसे अक्सर CD के रूप में संक्षिप्त किया जाता है, प्रथाओं का एक सेट है जिसमें कोड परिवर्तन स्वचालित रूप से एक स्वीकृति वातावरण में (या, निरंतर डिप्लॉयमेंट के मामले में, उत्पादन में) डेप्लॉये किए जाते हैं। CD में महत्वपूर्ण रूप से यह सुनिश्चित करने के लिए प्रक्रियाएं शामिल हैं कि डिप्लॉयमेंट से पहले सॉफ़्टवेयर का पर्याप्त रूप से परीक्षण किया गया है और यह रोलबैक परिवर्तनों का एक तरीका प्रदान करता है, यदि आवश्यक समझा जाता है तो । निरंतर डिलीवरी की दिशा में पहला कदम निरंतर इंटीग्रेशन (CI) है (यानी, परिवर्तनों को परीक्षण और डेप्लॉये करने से पहले, उन्हें साफ-सुथरे रूप से मर्ज करना होगा)।
```
Comment (verma-kunal) [inline content/hi/continuous_delivery.md:8]:
```suggestion
निरंतर डिलीवरी, जिसे अक्सर CD के रूप में संक्षिप्त किया जाता है, प्रथाओं का एक समूह है जिसमें कोड परिवर्तन स्वचालित रूप से एक स्वीकृति वातावरण में (या, निरंतर डिप्लॉयमेंट के मामले में, उत्पादन में) डेप्लॉये किए जाते हैं। CD में महत्वपूर्ण रूप से यह सुनिश्चित करने के लिए प्रक्रियाएं शामिल हैं कि डिप्लॉयमेंट से पहले सॉफ़्टवेयर का पर्याप्त रूप से परीक्षण किया गया है और यह रोलबैक परिवर्तनों का एक तरीका प्रदान करता है, यदि आवश्यक समझा जाता है तो । निरंतर डिलीवरी की दिशा में पहला कदम निरंतर इंटीग्रेशन (CI) है (यानी, परिवर्तनों को परीक्षण और डेप्लॉये करने से पहले, उन्हें साफ-सुथरे रूप से मर्ज करना होगा)। 
```

File: content/hi/continuous_delivery.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: निरंतर डिलीवरी (CD)
+status: Completed
+category: concept
+---
```
Comment (anubha-v-ardhan) [inline content/hi/continuous_delivery.md:5]:
```suggestion
---
title: निरंतर डिलीवरी (CD)
status: Completed
category: अवधारणा
---
```

Comment (anubha-v-ardhan) [reply content/hi/continuous_delivery.md:5]: we can safely localize the category label. Refer [spreadsheet](https://docs.google.com/spreadsheets/d/1MkZXrXre7dFVFmrRIHgHhNOYFEYcJkRaIWc3Dp8M-TQ/edit#gid=0)

File: content/hi/continuous_delivery.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## यह क्या है 
+निरंतर डिलीवरी, जिसे अक्सर CD के रूप में संक्षिप्त किया जाता है, प्रथाओं का एक समूह है जिसमें कोड परिवर्तन स्वचालित रूप से एक स्वीकृति वातावरण में (या, निरंतर डिप्लॉयमेंट के मामले में, उत्पादन में) डेप्लॉये किए जाते हैं। CD में महत्वपूर्ण रूप से यह सुनिश्चित करने के लिए प्रक्रियाएं शामिल हैं कि डिप्लॉयमेंट से पहले सॉफ़्टवेयर का पर्याप्त रूप से परीक्षण किया गया है और यह रोलबैक परिवर्तनों का एक तरीका प्रदान करता है, यदि आवश्यक समझा जाता है तो । निरंतर डिलीवरी की दिशा में पहला कदम निरंतर इंटीग्रेशन (CI) है (यानी, परिवर्तनों को परीक्षण और डेप्लॉये करने से पहले, उन्हें साफ-सुथरे रूप से मर्ज करना होगा)।
```
Comment (anubha-v-ardhan) [inline content/hi/continuous_delivery.md:8]:
```suggestion
निरंतर डिलीवरी, जिसे अक्सर CD के रूप में संक्षिप्त किया जाता है, प्रथाओं का एक समूह है जिसमें कोड परिवर्तन स्वचालित रूप से एक स्वीकृति वातावरण में (या, निरंतर डिप्लॉयमेंट के मामले में, उत्पादन में) डेप्लॉय किए जाते हैं। CD में महत्वपूर्ण रूप से यह सुनिश्चित करने के लिए प्रक्रियाएं शामिल हैं कि डिप्लॉयमेंट से पहले सॉफ़्टवेयर का पर्याप्त रूप से परीक्षण किया गया है और यदि आवश्यक समझा जाता है तो यह रोलबैक परिवर्तनों का एक तरीका प्रदान करता है। निरंतर डिलीवरी की दिशा में पहला कदम निरंतर इंटीग्रेशन (CI) है (अर्थात, परिवर्तनों को परीक्षण और डेप्लॉय करने से पहले, उन्हें साफ-सुथरे रूप से मर्ज करना)। 
```

File: content/hi/continuous_delivery.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+title: निरंतर डिलीवरी (CD)
+status: Completed
+category: concept
+---
+
+## यह क्या है 
+निरंतर डिलीवरी, जिसे अक्सर CD के रूप में संक्षिप्त किया जाता है, प्रथाओं का एक समूह है जिसमें कोड परिवर्तन स्वचालित रूप से एक स्वीकृति वातावरण में (या, निरंतर डिप्लॉयमेंट के मामले में, उत्पादन में) डेप्लॉये किए जाते हैं। CD में महत्वपूर्ण रूप से यह सुनिश्चित करने के लिए प्रक्रियाएं शामिल हैं कि डिप्लॉयमेंट से पहले सॉफ़्टवेयर का पर्याप्त रूप से परीक्षण किया गया है और यह रोलबैक परिवर्तनों का एक तरीका प्रदान करता है, यदि आवश्यक समझा जाता है तो । निरंतर डिलीवरी की दिशा में पहला कदम निरंतर इंटीग्रेशन (CI) है (यानी, परिवर्तनों को परीक्षण और डेप्लॉये करने से पहले, उन्हें साफ-सुथरे रूप से मर्ज करना होगा)। 
+
+## समस्या 
+[विश्वसनीय](/reliability/) उपदटेस को बड़े पैमाने पर डेप्लॉय करना एक समस्या बन जाता है। आदर्श रूप से, हम अंतिम-उपयोगकर्ताओं को बेहतर मूल्य प्रदान करने के लिए अधिक बार डेप्लॉय करेंगे। हालाँकि, इसे मैन्युअल रूप से करने से प्रत्येक परिवर्तन के लिए, अधिक खर्चा हो जाता हैं। ऐतिहासिक रूप से, इन खर्चो से बचने के लिए, संगठनों ने कम बार जारी किया है, एक बार में अधिक परिवर्तन लागू किए हैं और कुछ गलत होने का जोखिम बढ़ा दिया है।
```
Comment (anubha-v-ardhan) [inline content/hi/continuous_delivery.md:11]:
```suggestion
[विश्वसनीय](/reliability/) अपडेट को बड़े पैमाने पर डेप्लॉय करना एक समस्या बन जाता है। आदर्श रूप से, हम अंतिम-उपयोगकर्ताओं को बेहतर मूल्य प्रदान करने के लिए अधिक बार डेप्लॉय करते हैं। हालाँकि, इसे मैन्युअल रूप से करने से प्रत्येक परिवर्तन के लिए, अधिक खर्च होता हैं। ऐतिहासिक रूप से, इन खर्चो से बचने के लिए, संगठनों ने कम बार रिलीज़ करते है, एक बार में अधिक परिवर्तन डेप्लॉय करने पर कुछ गलत होने का जोखिम बढ़ जाता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you
Please go through review comments and commit them if you agree. Feel free to discuss if you find some inaccuracy or feel like a change is unnecessary

File: content/hi/continuous_delivery.md (hunk 2 above)
Comment (verma-kunal) [reply content/hi/continuous_delivery.md:5]: Sure 👍

**Review by anubha-v-ardhan — APPROVED** [review]
Thank you @verma-kunal

**Review by jayesh-srivastava — APPROVED** [review]

**Pull request merged**